packages/v4/src/content/developer-resources/accessibility-development.md

```diff
@@ -98,3 +98,19 @@ The recommended interaction pattern for the modal components like the modal or p
 - For screen reader users, the other contents on the page should be hidden from the screen reader.
 
 Based on the [screen reader / browser combinations](https://www.patternfly.org/v4/get-started/accessibility-guide#testing) we use for testing, we recommend applying `aria-hidden="true"` to the parent wrapping element of the page contents. Note that the modal element of the component must not be a descendent of this element with `aria-hidden="true"` and should be included as a sibling to this element.
+
+### Scrollable elements
+Any elements with horizontal or vertical scroll need to be accessible via keyboard. It may be necessary to ensure every 
+container with an overflow rule and content which could possibly grow larger than its parent's height to have 
+`tabindex=0`. This will ensure that parent can capture browser focus and be scrolled using the arrow keys. Some browsers
```

Comment:
When we add `tabIndex="0"` to an inactive element, like a div container for example, we turn this element into an interactive element - for interactive elements, it's important to ensure a well crafted accessible name exist.

If we only add tabIndex, and do not also ensure an accessible name is explicitly set, screen readers may not treat this content as expected

The main issue is that the container element with non-negative tabIndex will now be registered as an element in the form controls rotor menu, but the label for this element will be driven by whatever text content exist inside the container, which is variable and probably not suitable to use as a single label to name/describe the region's purpose.

That said, even if you're just using standard keyboard nav or screen reader cursor nav (and not rotor) the effect is the same.

Could we just also mention that when adding tabIndex in this scenario, it would also be good to add an aria-label that describes what the element is? I think that would do it..